docs/source/intervention_models/insecticide_treated_nets/index.rst

```diff
@@ -0,0 +1,241 @@
+    - 2008
+    - 0.60
+    - ITN use among pregnant women positively associated with household wealth index
+  * - [Yitayew-et-al-2018]
+    - 226 pregnant women who attended antenatal clinics at the Adis Zemen hospital
+    - 2018
+    - 0.708
+    - ITN use positively associated with maternal education and maternal age
+  * - [Fuge-et-al-2015]
+    - 398 pregnant women in Hadiya Zone, Shashogo District, Ethiopia
+    - 2014
+    - 0.082
+    - 
+  * - [Graves-et-al-2011]
+    - X
+    - 2006-2007
+    - X
+    - 
+  * - [Negash-et-al-2012]
+    - Six districts in Afar National Regional State in northeastern Ethiopia
+    - 2008
+    - 0.70
+    - Observed use (reported use was 79.1%)
+  * - [Ouedraogo-et-al-2019]
+    - 2784 women who had pregnancy outcome in the year preceding the survey
+    - 2016
+    - 0.264
+    - reported "always" sleeping under the net during their pregnancy. ITN use was positively associated with attending at least one ANC visit.
+
+.. todo::
+
+  Determine baseline coverage value to be used for Ethiopia, either by meta-analysis or by selection of large/representative recent study
+
+Vivarium Modeling Strategy
+--------------------------
+
+.. list-table:: Modeled Outcomes
+  :header-rows: 1
+
+  * - Outcome
+    - Outcome type
+    - Outcome ID
+    - Affected measure
+    - Effect size measure
+    - Effect size
+    - Note
+  * - Birthweight
+    - GBD risk exposure
+    - 339
+    - Population mean birthweight
+    - Mean difference
+    - +33 grams (95% CI: 5, 62)
+    - 
+
+Birthweight
++++++++++++++++++++++
+
+The ITN intervention affects child birthweight exposures, :ref:`which are documented here <2019_risk_exposure_lbwsg>`. The intervention should result in an **additive change to a simulant's continuous birthweight exposure value at birth (or upon initialization into the early or late neonatal age groups).** We assume there is no corresponding change in a simulant's gestational age exposure value at birth.
+
+.. list-table:: ITN effect on birthweight restrictions
+  :header-rows: 1
+
+  * - Restriction
+    - Value
+    - Note
+  * - Male only
+    - False
+    - 
+  * - Female only
+    - False
+    - 
+  * - Age group start
+    - Birth
+    - 
+  * - Age group end
+    - Late neonatal
+    - 
+  * - Other
+    - 
+    - 
+
+.. list-table:: ITN and Birthweight Effect Sizes
+  :header-rows: 1
+
+  * - Population
+    - Effect size
+    - Note
+  * - Pregnant women (overall)
+    - +33 grams (95% CI: 5, 62)
+    - [Gamble-et-al-2007]_
+  * - Pregnant women in first or second pregnancy
+    - +55 (95% CI: 21, 88)
+    - [Gamble-et-al-2007]_
+  * - Pregnant women in third or later pregnancy
+    - -20 (95% CI: -74, 33)
+    - [Gamble-et-al-2007]_
+
+.. note::
+
+  While there is evidence for effect modification of ITN on birthweight by maternal parity, we will model the overall effect until a maternal parity model is developed if/when needed
```

Comment:
Since we are not modeling household structure, we could potentially assign a birth parity value to each simulant on initialization, selected from the empirical distribution from the 2019 Ethiopia DHS